packages/web-components/src/components/va-search-input/va-search-input.tsx

```diff
@@ -439,7 +452,7 @@ export class VaSearchInput {
       return (
         <Host onBlur={handleBlur}>
           <form class={formClasses} role="search">
-            <label class="usa-sr-only" htmlFor="search-field">{label}</label>
+            <label class="usa-sr-only kokokok" htmlFor="search-field">{label}</label>
```

Comment:
Is "kokokok" just a leftover thing from testing?

Reply:
Whoops XD